pkg/cmd/pulumi/new_acceptance_test.go

```diff
@@ -64,6 +64,34 @@ func TestCreatingStackWithArgsSpecifiedName(t *testing.T) {
 	removeStack(t, tempdir, stackName)
 }
 
+//nolint:paralleltest // changes directory for process
+func TestCreatingStackWithNumericName(t *testing.T) {
+	tempdir, _ := os.MkdirTemp("", "test-env")
+	defer os.RemoveAll(tempdir)
```

Comment:
Use `t.TempDir()` here to have `testing.T` manage this for you.
```suggestion
	tempdir := t.TempDir()
```